### DISCUSSION TITLE
Skyline has come so far

### DISCUSSION THREAD

**Discussion by Joemama45** [skyline-emu/skyline#1387] [General]
Wow, I have been using this emulator on my device and I'm impressed with it. Never in my lifetime I would ever hear about a switch emulator on a moblie device, all I can say is, I'm hyped for every update the team releases keep up the great work!

**Comment by Carnell89** [comment]
How do you download the update from here

**Reply by Joemama45** [reply]
You have to go to the actions sections, and find the most recent update, then click on it
